electron/app/services/log-service.ts

```diff
@@ -7,6 +7,14 @@ export class LogService {
 
     async createOrUpdateLog(logAttributes: Partial<Log>): Promise<Log> {
         let log: Log = null;
+        // Skip known/unfixable errors
+        if (logAttributes.message.includes("'endDate' of null")) {
+            return log;
+        }
+        if (logAttributes.message.includes('getaddrinfo ENOTFOUND hasura.gitstart.com')) {
+            logAttributes.message =
+                'GitStart DevTime was disconnected. Please make sure you are connected to the internet.';
+        }
```

Comment:
Not recording/showing these errors to the user as they seem to be one time flukes (e.g. not logged in when quitting app).